### PR TITLE
migrate to Visual Studio RC

### PR DESCRIPTION
**Pull request by SergKanz**
Migrate develop branch to Visual Studio 2015 RC. Also updated application template to test integration with the default template from that Visual Studio

### CODE REVIEW

**Comment by olegsych** [comment]
My head just went :boom:. Can we simplify the functional tests now? :cherries:

File: src/Microsoft.ApplicationInsights.AspNet/project.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "title": "Application Insights for ASP.NET 5 Web Applications",
     "description": "Application Insights for ASP.NET 5 web applications. See http://go.microsoft.com/fwlink/?LinkID=510752 for more information.",
     "authors": [ "Microsoft" ],
-    "version": "0.30.0.1-beta",
+    "version": "0.32.0-beta4",
```
Comment (SergKanz) [inline src/Microsoft.ApplicationInsights.AspNet/project.json]: this file is interesting to look at

File: test/FunctionalTestUtils/project.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ﻿{
     "version": "1.0.0-*",
```
Comment (SergKanz) [inline test/FunctionalTestUtils/project.json]: this file is interesting to look at

File: test/Mvc6Framework45.FunctionalTests/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -1,49 +1,83 @@
 ﻿@using Microsoft.Framework.DependencyInjection
 @inject Microsoft.ApplicationInsights.DataContracts.RequestTelemetry RequestTelemetry 
+@inject IOptions<AppSettings> AppSettings
```
Comment (SergKanz) [inline test/Mvc6Framework45.FunctionalTests/Views/Shared/_Layout.cshtml]: AI injection here

File: test/Mvc6Framework45.FunctionalTests/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -1,49 +1,83 @@
...
+        <environment names="Staging,Production">
+            <link rel="stylesheet" href="//ajax.aspnetcdn.com/ajax/bootstrap/3.0.0/css/bootstrap.min.css"
+                  asp-fallback-href="~/lib/bootstrap/css/bootstrap.min.css"
+                  asp-fallback-test-class="hidden" asp-fallback-test-property="visibility" asp-fallback-test-value="hidden" />
+            <link rel="stylesheet" href="//ajax.aspnetcdn.com/ajax/bootstrap-touch-carousel/0.8.0/css/bootstrap-touch-carousel.css"
+                  asp-fallback-href="~/lib/bootstrap-touch-carousel/css/bootstrap-touch-carousel.css"
+                  asp-fallback-test-class="carousel-caption" asp-fallback-test-property="display" asp-fallback-test-value="none" />
+            <link rel="stylesheet" href="~/css/site.css" />
+        </environment>
 	@Html.ApplicationInsightsJavaScriptSnippet(RequestTelemetry.Context.InstrumentationKey);
```
Comment (SergKanz) [inline test/Mvc6Framework45.FunctionalTests/Views/Shared/_Layout.cshtml]: and here

File: test/Mvc6Framework45.FunctionalTests/config.json [hunk 5]
```diff
@@ -1,15 +1,13 @@
...
-	}
+  "AppSettings": {
+    "SiteTitle": "Mvc6Framework45.FunctionalTests"
+  },
+  "Data": {
+    "DefaultConnection": {
+      "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet5-Mvc6Framework45.FunctionalTests-42f45116-363a-40d3-9c44-bc58885d6e96;Trusted_Connection=True;MultipleActiveResultSets=true"
+    }
+  },
+  "ApplicationInsights": {
```
Comment (SergKanz) [inline test/Mvc6Framework45.FunctionalTests/config.json]: and here

File: test/Mvc6Framework45.FunctionalTests/Startup.cs [hunk 6]
```diff
@@ -1,49 +1,92 @@
...
         }
 
         public IConfiguration Configuration { get; set; }
 
-        // This method gets called by the runtime.
+        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
             // Add Application Insights services to the services container.
             services.AddApplicationInsightsTelemetry(Configuration);
```
Comment (SergKanz) [inline test/Mvc6Framework45.FunctionalTests/Startup.cs]: application Insights here and further in this file

**Comment by SergKanz** [comment]
most of this pull requests is just taking new application template in VS 2015 RC and using it instead of CTP6 template. I marked all integration points. Whole purpose of this test is to ensure that AI works end to end for default application. Everything else should be covered by unit tests as you mentioned in [issue #19](https://github.com/Microsoft/ApplicationInsights-aspnetv5/issues/19)

**Comment by olegsych** [comment]
:shipit:

**Pull request merged**